packages/amplication-client/src/Project/ArchitectureConsole/ModelOrganizer.tsx

```diff
@@ -349,6 +359,7 @@ export default function ModelOrganizer({ restrictedMode = false }: Props) {
                 onNodeDragStop={onNodeDragStop}
                 onEdgesChange={onEdgesChange}
                 connectionMode={ConnectionMode.Loose}
+                onNodeClick={onNodeClick}
```

Comment:
Is this only effective for the entity nodes and not the resource nodes?

Reply:
@mulygottlieb is effective for Both of them, but in the callback function, there is a check that prevents the process from continuing,